vpd-manager/include/ipz_parser.hpp

```diff
@@ -106,6 +106,13 @@ class IpzVpdParser : public ParserInterface
      */
     int writeKeywordOnHardware(const types::WriteVpdParams i_paramsToWriteData);
 
+    /**
+     * @brief API to perform sanity check on EEPROM.
+     *
```

Comment:
Please add a brief description in the doxygen to explain what does sanity check means.